src/bitcoin-cli.cpp

```diff
@@ -604,6 +618,8 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "           \".\" (to)   - we selected the peer as a high-bandwidth peer\n"
         "           \"*\" (from) - the peer selected us as a high-bandwidth peer\n"
         "  age      Duration of connection to the peer, in minutes\n"
+        "  addrl    Total number of addresses dropped due to rate limiting\n"
```

Comment:
because 39343849904ea53bb3e9c66521cfccdfd602f46e introduces the new fields and 22fa9112cecb3cfe8a299b7bba76267282e6d76f puts them to proper use by moving them into their new proper level, its kind of like 'introduce then use'.

Total non-blocker, but I think you could combine the first two commits into something like:

`cli: introduce netinfo level 5 with addr_processed and addr_rate_limited stats`

Reply:
Dropped the additional level (thanks!)